docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -4,116 +4,159 @@ title: Cellular
 
-To use this module you will need to connect the **Meadow F7v2 Feather** `D00` and `D01` pins to the `TX` and `RX` click board pins, respectively, to establish the data communication between them.
+## Choosing a SIM Card
 
-#### Power-up and supply pins
+To enable **Cat-M1** (LTE-M or eMTC) or **NB-IoT** network modes, a specialized **M2M** (Machine-to-Machine) SIM card is required, distinct from the standard SIM cards used in cellphones. However, for **GSM/GPRS 2G** connections, a standard SIM card can generally be used.
 
-Also, you need to connect the **Meadow F7v2 Feather** `D10` pin to the `EN` **NimbeLink Skywire click board** pin. Additionally, connect the `3.3V`, `5V`, and `GND` pins from the **Meadow F7v2 Feather** to their corresponding pins on the click board.  If you are using another click board for the **BG95-M3** module, you need to connect the `D10` pin to the equivalent power-up pin.
+# Enabling Cellular on your Meadow Application
 
-#### Attaching an antenna
+Using Cellular on Meadow, you will need to do three things:
 
-Finally, connect an LTE antenna (Rubber ducky or Dome) with the `X1` click board IPX connector, aiming for a preferred gain of 5 dBi (recommended) while ensuring a minimum gain of 2 dBi (required), and insert an **M2M** SIM card into the cell module.
+## Adding a cell.config.yaml file
 
-![Quectel BG95-M3/NimbeLink Skywire click board setup](images/bg95.jpeg){: .center-image :standalone}
+Create a **cell.config.yaml** file, set the `Copy To Output` property to `Copy always` or `Copy if newer`, and fill out the values under the `Settings` section like APN, Module, and a additional optional pins to ensure a proper connection with your internet provider. Here's what a cell config file looks like with the required and optional fields:
 
-<p align="center" style="font-size: smaller;">Quectel BG95-M3 click board with Meadow F7v2 Feather setup</p>
+```yaml
+Settings:
+    APN: YOUR-APN         # (required) Access Point Name
+    Module: BG95M3        # (required) Module model (BG770A, BG95M3 or M95)
+    User: USER            # (optional) APN user 
+    Password: PASSWORD    # (optional) APN password
+    Operator: 00000       # (optional) Carrier numeric operator code
+    Mode: CATM1           # (optional) Network mode (CATM1, NBIOT or GSM)
+    Interface: /dev/ttyS1 # (optional) Serial interface 
+                          # UART1 (COM1) = /dev/ttyS0 
+                          # UART4 (COM4) = /dev/ttyS1 (default), 
+                          # UART6 = /dev/ttyS3) 
+    TurnOnPin: D10        # (optional) Enable pin to turn the module on/off. 
+                          # Default value is Meadow Pin name D10
```

Comment:
We should add the `ScanMode` to that:
`        ScanMode: false # Activate the cell network scanning mode (optional)`